views/chitter/index.erb

```diff
@@ -0,0 +1,39 @@
+<html>
+
+<style>
+body {
+    background-color: #c4c4c4;
+  
+}
+
+.text {
+    position: absolute;
```

Comment:
The site looks really good, all this CSS makes it look great. I wonder whether you could have all your css on a style sheet inside a public folder, that SInatra would go and find before running the program?